src/index.css

```diff
@@ -1,3 +1,13 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+}
+
+#root {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 100vw;
```

Comment:
I think it is not recommended to style the `#root` element, by react best practice standards. You could create a parent element in your `App.js` to have a similar effect.